scripts/4.covariates.py

```diff
@@ -21,60 +19,55 @@
 
 path = os.path.join('data', 'samples.tsv')
 covariates_df = pd.read_table(path, index_col=0)
+covariates_df = covariates_df.rename(columns={'organ_of_origin': 'organ'})
 covariates_df.head(10)
 
 
 # Let's get a sense of how much missing data there is.
 
 # In[3]:
 
-print('Total number of samples: %d' % len(covariates_df))
+print('Total number of samples: {:,}'.format(len(covariates_df)))
 print('Number of nulls in each column:')
 covariates_df.isnull().sum(axis=0)
 
 
-# Now count the number of types of each column. We'll use Python's set, and therefore must exclude null values.
-
 # In[4]:
 
-categorical_variables = ['patient_id', 'sample_type', 'disease', 'organ_of_origin', 'gender', 'dead', 'recurred']
-for variable in categorical_variables:
-    not_null = covariates_df[variable].notnull()
-    count = len(set(covariates_df.loc[not_null, variable]))
-    print('Number of types in category %s: %d' % (variable, count))
+# Specify which variables are categorical
+categorical_variables = ['acronym', 'organ', 'gender', 'dead', 'recurred']
```

Comment:
'dead' should be 'vital_status' - this is the proper way to describe the variable with dead = 1 and alive = 0